src/main.ts

```diff
@@ -158,10 +158,21 @@ async function deactivateRevision(params: any) {
   // Check traffic weight of the target revision
   if (targetRevisions.length > 0 && targetRevisions.reduce((prev: number, curr: any) => prev + curr.weight, 0) !== 0)
     throw new Error(`Traffic weight of revision ${revisionName} under container app ${containerAppName} is not 0. Set 0 to the traffic weight of the revision before deactivation.`);
-
+ 　　
   console.log("Deactivation Step Started");
   await client.containerAppsRevisions.deactivateRevision(resourceGroup, containerAppName, revisionName);
-  console.log("Deactivation Step Succeeded");
+  
+  // check if revision's status is deactived
+  const deactiveRevision = await client.containerAppsRevisions.getRevision(
+    resourceGroup,
+    containerAppName,
+    revisionName
+  )
+  if(deactiveRevision.active) {
```

Comment:
If true, it may be better to write a successful process and if false, an error.  And, how about adding `core.setFailed` if the process fails?

Reply:
Agreed. I recommend throwing an exception and jump to `catch` block where `core.setFailed` is used in .
https://github.com/Azure/aca-review-apps/pull/54#discussion_r975965652

Reply:
Sorry, I thought it was the main function that did this, but there is a deactivateRevision function that does the error handling. It would be cleaner to return "error" as the return value. Either is fine. 😄

Reply:
For now throw an error if revision cannot be deactivated is a good idear.
Change is done.